Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -11,8 +11,9 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     private let controller = GutenbergEditorController()
     private let timestampInit = CFAbsoluteTimeGetCurrent()
     private let service: EditorService
-    private let siteURL: String
+    private let siteAPIURL: String
     private let authToken: String
+    private let siteID: NSNumber?
```

Comment:
Can we use a scalar type here? 

Ideally we'd be more specific than `NSNumber` for stuff like this

Related – do we expect that the `siteID` will be optional?

Reply:
Thanks for the suggestion, I ended up removing `siteID` in https://github.com/wordpress-mobile/GutenbergKit/commit/f268246891ae3a76be68c72a409e5f5ee64d40c2#diff-8d684f85d6faf91b7cef2e39f371584ec64f48bee75eec04292272b85272a5d4

>Related – do we expect that the siteID will be optional – I'm thinking that the siteApiRoot would have all the information we need, but WDYT?

I tried to unify the information we need in `siteApiRoot` but for sites that use API URLs like `https://public-api.wordpress.com/wp/v2/sites/mysite.wordpress.com` don't work well with the editor's internal API namespaces, where the REST namespace is appended to the root API URL ending up with URLs like `/wp/v2/sites/mysite.wordpress.com/wp/v2/media` or in other cases `/wp/v2/sites/mysite.wordpress.com/oembed/1.0/proxy`.

As a potential solution, I added `siteApiNamespace` that will have a value like `sites/mysite.wordpress.com` (if it's a .com site) so it's easier to format the final URL.